CmsManager/CmsManagerInterface.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DashboardBundle\CmsManager;
+
+use Sonata\BlockBundle\Model\BlockInterface;
+use Sonata\DashboardBundle\Model\DashboardInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * The CmsManagerInterface class is in charge of retrieving the correct dashboard.
```

Comment:
@greg0ire it's managing content for dashboards, so "cms", but i understand your concern.
BTW, can you explain me how i can use the styleCI patch to modify individually the 5 first commits ? long learning journey on git ...